examples/kubernetes/vttablet-down.sh

```diff
@@ -21,18 +21,22 @@ num_shards=`echo $SHARDS | tr "," " " | wc -w`
 uid_base=$UID_BASE
 
 for shard in `seq 1 $num_shards`; do
-  for uid_index in `seq 0 $(($TABLETS_PER_SHARD-1))`; do
-    uid=$[$uid_base + $uid_index]
-    printf -v alias '%s-%010d' $cell $uid
-
-    if [ -n "$server" ]; then
-      echo "Removing tablet $alias from Vitess topology..."
-      vtctlclient -server $server ScrapTablet -force $alias
-      vtctlclient -server $server DeleteTablet $alias
-    fi
-
-    echo "Deleting pod for tablet $alias..."
-    $KUBECTL delete pod vttablet-$uid
+  cell_index=100000000
```

Comment:
Yeah I'll try that, I think I can also get rid of the changes to the yaml files since the UIDs are unique anyway